ui/page/root/home_page.go

```diff
@@ -810,3 +821,33 @@ func (hp *HomePage) CalculateAssetsUSDBalance() {
 		hp.ParentWindow().Reload()
 	}
 }
+
+// initializeDEX will initialize hp.dexCore if it has not been initialized
+// before. It's a no-op if hp.dexCore has been initialized.
+func (hp *HomePage) ensureDEXIsReady() {
+	if hp.dexCore != nil && hp.dexCoreRunning {
+		return // nothing to do
+	}
+
+	logger, logCloser, err := dexcore.NewDexLogger(filepath.Dir(hp.WL.AssetsManager.LogFile()), hp.WL.AssetsManager.GetLogLevels(), hp.WL.AssetsManager.MaxLogZips)
```

Comment:
Isn't `AssetsManager.LogFile()` currently in use by another log rotator?

Reply:
We need just the dir `filepath.Dir(hp.WL.AssetsManager.LogFile())`.